build.ps1

```diff
@@ -271,12 +243,12 @@ if ($Configuration) { $cakeArguments += "-configuration=$Configuration" }
 if ($Verbosity) { $cakeArguments += "-verbosity=$Verbosity" }
 if ($ShowDescription) { $cakeArguments += "-showdescription" }
 if ($DryRun) { $cakeArguments += "-dryrun" }
-if ($Experimental) { $cakeArguments += "-experimental" }
-if ($Mono) { $cakeArguments += "-mono" }
 $cakeArguments += $ScriptArgs
 
+# Dump environment variables to aid in debugging
+gci env: | findstr /v /i key | findstr /v /i secret
```

Comment:
This is the one change I made to help us with future debugging

Reply:
I'm fine with this change...

It will likely be LOST when we update to a newer version of `build.ps1` but since it's for debugging only there is no risk involved.

Reply:
Security related question:

Do any environment variables have security related information that should NOT be in the log file?

If so we should NOT do this.

Reply:
Discussing offline